CONTRIBUTING.md

```diff
@@ -93,5 +93,5 @@ go test -v
 # run single test
 go test -v <path> -run <TestSuite> -testify.m <TestSpercificTaskName>
 # example:
-go test -v github.com/uber/cadence/common/persistence -run TestCassandraPersistenceSuite -testify.m TestPersistenceStartWorkflow
+go test -v github.com/uber/cadence/common/persistence/persistence-tests -run TestVisibilitySamplingSuite -testify.m TestListClosedWorkflowExecutions
```

Comment:
Can we keep this unchanged?

Reply:
I guess, but old one isn't working, there is no TestCassandraPersistenceSuite in this directory. Do you still want me to revert?

Reply:
@meiliang86 ping